src/app/features/aMMs/reducer.ts

```diff
@@ -40,45 +36,13 @@ export const slice = createSlice({
         state.aMMsLoadedState = 'failed';
       })
       .addCase(initialiseAMMsThunk.fulfilled, (state, action) => {
+        const { amms, error } = action.payload as Awaited<ReturnType<typeof getAMMs>>;
```

Comment:
This is kinda antipattern. Let's improve on this. If error we should of ended up in the failed statement